components/gitpod-db/leeway.Dockerfile

```diff
@@ -2,19 +2,20 @@
 # Licensed under the GNU Affero General Public License (AGPL).
 # See License-AGPL.txt in the project root for license information.
 
-FROM node:12.18.3-slim as builder
+FROM node:12.18.3 as builder
 COPY components-gitpod-db--migrations /installer/
 
 WORKDIR /app
 RUN /installer/install.sh
 
 
-FROM node:12.18.3-slim
+FROM node:12.18.3
 RUN mkdir /home/jenkins && chown -R 10000 /home/jenkins
 COPY --chown=10000:10000 --from=builder /app /app/
 WORKDIR /app/node_modules/@gitpod/gitpod-db
 
-RUN echo "#!/bin/bash"                                                            >> /app/migrate.sh && \
-    echo "yarn --cwd /app/node_modules/@gitpod/gitpod-db run wait-for-db"        >> /app/migrate.sh && \
-    echo "yarn --cwd /app/node_modules/@gitpod/gitpod-db typeorm migrations:run" >> /app/migrate.sh && \
-    chmod +x /app/migrate.sh
\ No newline at end of file
+RUN wget https://dl.google.com/cloudsql/cloud_sql_proxy.linux.amd64 -O /bin/cloud_sql_proxy \
```

Comment:
This should be done before line 14 - there's no need to invalidate the cloud_sql_proxy just because the migrations changed.

Reply:
done